crypto-core/src/main/java/com/palantir/crypto2/io/ApacheCtrDecryptingSeekableInput.java

```diff
@@ -67,8 +67,8 @@ public void close() throws IOException {
     }
 
     private static final class InputAdapter implements Input {
-
         private SeekableInput input;
+        private byte[] readBuffer = new byte[BUFFER_SIZE];
```

Comment:
final

Reply:
yep, makes sense if we know the buffer is constant size